src/server/ua_subscription_events.h

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2018 (c) Ari Breitkreuz, fortiss GmbH
+ */
+
+#ifndef UA_SUBSCRIPTION_EVENTS_H_
+#define UA_SUBSCRIPTION_EVENTS_H_
+
+#ifdef UA_ENABLE_EVENTS
+
+#include "ua_subscription.h"
+
+#define UA_NS0ID_SIMPLEOVERFLOWEVENTTYPE 4035
+
+/* generates a unique eventId */
+UA_StatusCode UA_Event_generateEventId(UA_Server *server, UA_ByteString *outId);
```

Comment:
Remove all the methods, which are only used in the `ua_subscription_events.c` file from this header and make them static there. Keep them hidden from the rest of the code if they are not used in any other compilation unit.
Don't forget to copy the documentation into the .c file if you remove it here.

Reply:
You can keep these in the separate header file so we have a clean basis for future implementation